### PR TITLE
Cover bottlerocket cluster test.

### PR DESCRIPTION
**Pull request by bnapolitan**
Create cluster using bottlerocket configuration. Ensure integration/conformance tests pass across this cluster.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
       - run:
           name: Run the integration tests
           command: ./scripts/run-integration-tests.sh
-          no_output_timeout: 15m
+          no_output_timeout: 20m
```
Comment (mogren) [inline .circleci/config.yml:76]: Oh, it's that slow?

File: scripts/run-integration-tests.sh [hunk 2]
```diff
@@ -170,11 +184,15 @@ sed -i'.bak' "s,:$MANIFEST_IMAGE_VERSION,:$TEST_IMAGE_VERSION," "$TEST_CONFIG_PA
 sed -i'.bak' "s,602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init,$INIT_IMAGE_NAME," "$TEST_CONFIG_PATH"
 sed -i'.bak' "s,:$MANIFEST_IMAGE_VERSION,:$TEST_IMAGE_VERSION," "$TEST_CONFIG_PATH"
 
-if [[ $RUN_KOPS_TEST != true ]]; then
-    export KUBECONFIG=$KUBECONFIG_PATH
+if [[ $RUN_KOPS_TEST == true || $RUN_BOTTLEROCKET_TEST == true ]]; then
+    KUBECTL_PATH=kubectl
+    export KUBECONFIG=~/.kube/config
```
Comment (mogren) [inline scripts/run-integration-tests.sh:189]: Ok, so both Kops and eksctl needs these to be set.

File: .circleci/config.yml (hunk 1 above)
Comment (bnapolitan) [reply .circleci/config.yml:76]: eksctl can sometimes take longer than 15m to create the cluster and mng, so it did timeout once. Haven't seen any problems after moving it to 20m!

File: scripts/run-integration-tests.sh [hunk 3]
```diff
@@ -151,10 +154,21 @@ mkdir -p "$TEST_CLUSTER_DIR"
...
-if [[ "$PROVISION" == true && "$RUN_KOPS_TEST" == true ]]; then
-    up-kops-cluster
-elif [[ "$PROVISION" == true ]]; then
-    up-test-cluster
+if [[ "$PROVISION" == true ]]; then
+    START=$SECONDS
+    if [[ "$RUN_BOTTLEROCKET_TEST" == true ]]; then
+        echo "TESTING downloading eksctl"
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv -v /tmp/eksctl /usr/local/bin
```
Comment (jaypipes) [inline scripts/run-integration-tests.sh:162]: Please place the above code into an `ensure_eksctl` Bash function (similar to `ensure_ecr_repo` and `ensure_aws_k8s_tester` functions.

File: scripts/run-integration-tests.sh [hunk 4]
```diff
@@ -151,10 +154,21 @@ mkdir -p "$TEST_CLUSTER_DIR"
...
+        eksctl create cluster --config-file ./testdata/bottlerocket.yaml
+    elif [[ "$RUN_KOPS_TEST" == true ]]; then
+        up-kops-cluster
+    else
+        up-test-cluster
+    fi
+    UP_CLUSTER_DURATION=$((SECONDS - START))
+    echo "TIMELINE: Upping test cluster took $UP_CLUSTER_DURATION seconds."
+    __cluster_created=1
```
Comment (jaypipes) [inline scripts/run-integration-tests.sh:171]: You don't need to set `__cluster_created=1` above, since it's set on line 173 below.

**Pull request merged**